revaluate_client/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './Header.css';
+
+function Header(){
+    return(
+        <div>
+            <nav class = "headerBar">
```

Comment:
the react syntax is 'className' instead of 'class'

Reply:
ok i'll change it